src/server/middlewares/authentication.js

```diff
@@ -34,8 +34,17 @@ const isAdmin = (req, res, next) => {
   }
   res.status(403).send({ message: 'You are not an Authorised user' });
 };
+
+const isAssignee = (req, res, next) => {
+  const Assignee = 2;
+  if (res.locals.roleId === Assignee) {
+    return next();
+  }
+  res.status(403).send({ message: 'You are not an Authorised user' });
+};
+
```

Comment:
@mojo706 This function with `isAdmin` above are completely the same. Just make one function and call it something like `canViewIncidents` and also add JsDocs please. Then do:
```js
const canViewIncidents = (req, res, next) => {
  cont roleId = res.locals.roleId;
  if (roleId === 2 || roleId === 3) {
    return next();
  }
  res.status(403).send({ message: 'You are not an Authorised user' });
};
```